compiler/il/OMRSymbol.hpp

```diff
@@ -398,8 +393,32 @@ class OMR_EXTENSIBLE Symbol
    inline void setMemoryTypeShadowSymbol();
    inline bool isMemoryTypeShadowSymbol();
 
-   void setOrdered() { _flags.set(Ordered); }
-   bool isOrdered()  { return _flags.testAny(Ordered); }
+   enum MemoryOrdering
+      {
+      TransparentSemantics,
+      OptOpaqueSemantics,
```

Comment:
What is the significance of the `Opt` prefix in the name `OptOpaqueSemantics`?  I'm just wondering whether it could be changed to simply `OpaqueSemantics`